### DISCUSSION TITLE
Exporting comments

### DISCUSSION THREAD

**Discussion by kieranopat** [gavinr/github-csv-tools#50] [Q&A] [answered]
Hi Gavin! Thank you for this awesome tool. I am encountering an issue exporting the comments connected with the issues that I have pulled from my repository. I was able to download all attributes of all files (which is awesome!) but the `comments` field only contains a number from 0-X based on the # of comments on the issue--not the text of the comments themselves. I've done research on other solutions to export these comments but have had no luck...any ideas? Thank you!

**Comment by gavinr** [comment, marked as answer]
did you use the `--exportComments` switch?

**Reply by kieranopat** [reply]
my terminal was getting hung up while trying to run that command. restarted mac and will try again

**Reply by kieranopat** [reply]
figured out the issue...I just had to be patient! CSV output after about 7 mins. Thank you for this script! It is awesome!
